portfolio/src/main/webapp/style.css

```diff
@@ -111,25 +111,25 @@
 .dropdown-click:hover, .dropdown-click:focus {
   background-color: #6db4e2;
 }
-/* dropdown content (hidden by default) */
+/* Dropdown content hidden by default. */
 .dropdown-content {
   display: none;
   position: relative;
 }
-/* show the dropdown content */
+/* Shows previously hidden content. */
 .show {
   display:block;
 }
 
 /* gallery.html */
 
-/* gallery side nav bar */
+/* Gallery side nav bar. */
 .side-nav {
```

Comment:
Name vs Comment Consistency:

"Side Nav Bar Gallery" or change the class name to `side-nav-gallery`

or if this nav bar has more than just a gallery, or you might add more to it in the future, keep the name as `side-nav` but make the comment: "Side nav bar, containing the gallery"

Reply:
I'm changing the comment to "Side nav bar, contained in the gallery"